packages/rescript-relay-router/rollup.config.js

```diff
@@ -0,0 +1,34 @@
+import resolve from "@rollup/plugin-node-resolve";
+
+export default [
+  {
+    input: "cli/RescriptRelayRouterCli.mjs",
+    output: {
+      file: "cli/RescriptRelayRouterCli.bundle.mjs",
+      format: "esm",
+      name: "RescriptRelayRouterCli",
+    },
+    plugins: [resolve({ resolveOnly: [/^@rescript\/.*$/] })],
```

Comment:
Maybe we should bundle the ReScript stdlib stuff as well. It's not necessarily the case that what version the router was built with is the same as the version used locally in the project. So bundling those too wouldn't hurt. What do you think?

Reply:
You're right, we can do that easily so let's do it!